src/Test/Utilities/Portable/Roslyn.Test.Utilities.csproj

```diff
@@ -77,6 +77,9 @@
     <Compile Include="..\..\..\Compilers\Shared\DesktopAnalyzerAssemblyLoader.cs">
       <Link>DesktopAnalyzerAssemblyLoader.cs</Link>
     </Compile>
+    <Compile Include="..\..\..\Compilers\Shared\ShadowCopyAnalyzerAssemblyLoader.cs">
+      <Link>ShadowCopyAnalyzerAssemblyLoader.cs</Link>
```

Comment:
Nit: you can avoid the Link element here if you like. It's not actually needed unelss you want to change the file name. We should probably just clean all this up.